lms/static/js/instructor_dashboard/membership.js

```diff
@@ -966,11 +995,28 @@ such that the value can be defined later than this assignment (file load order).
                 authList = $opt.data('auth_list');
                 authList.$container.addClass('active');
                 authList.re_view();
+
+                // On Change update the Group Moderation list
+                if ($opt.text() === 'Discussion Group Community TA') {
+                    if (!authList.list_enabled) {
+                        authList.show_errors(gettext('This role requires a divided discussions scheme.'));
+                        authList.$('input[type="button"].add').addClass('is-disabled');
```

Comment:
Nit: Same comment about defining button outside of the if/else. One could also imagine creating a helper method that is shared between this code and the code up at line 172.